spec/configuration_spec.rb

```diff
@@ -7,7 +7,8 @@
   it { expect(instance.redirect_limit).to         eq(3) }
   it { expect(instance.user_agent).to             eq('link_thumbnailer') }
   it { expect(instance.verify_ssl).to             eq(true) }
-  it { expect(instance.http_timeout).to           eq(5) }
+  it { expect(instance.http_open_timeout).to      eq(5) }
+  it { expect(instance.http_read_timeout).to      eq(5) }
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.